docs/proposals/hpa/cronfederatedhpa.md

```diff
@@ -0,0 +1,347 @@
+---
+title: CronFederatedHPA
+authors:
+- "@jwcesign"
+reviewers:
+- "TDB"
+approvers:
+- "TDB"
+
+creation-date: 2023-06-06
+
+---
+
+# CronFederatedHPA
+## Summary
+
+<!--
+This section is incredibly important for producing high-quality, user-focused
+documentation such as release notes or a development roadmap. 
+
+A good summary is probably at least a paragraph in length.
+-->
+
+For some certain predictable load change scenarios, for example Black Friday sale and Singles' Day sale, there will be sharp load peaks. With standard K8s HPA, it needs time to scale up the replicas, which may not be fast enough to handle sharp load peaks, resulting in service unavailability.
+
+To solve this problem, there are already some approaches in single cluster, called `CronHPA`, which can scale up/down the workloads at specific time. Cluster administrators can scale up the workloads in advance with `CronHPA`, to meet the requirements of the sharp load peaks. So this proposal proposes `CronFederatedHPA` to meet the the same requirements in multi-cluster scenario.
+
+## Motivation
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users.
+-->
+
+The scaling speed is not fast enough to handle sharp load peaks, causing service unavailability, which is unacceptable. So the cluster administrators want to scale up/down the workloads in advance to meet the sharp load peaks in multi-cluster scenario, ensure the service is always available. This is critical in some special days, such as Black Friday sale and Singles' Day sale.
+
+### Goals
+
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+1. Define the `CronFederatedHPA` API, to scale the workloads in specific time. 
+1. Support both `FederatedHPA` and workloads(`Deployment`, `StatefulSet` or other).
```

Comment:
`other` what? We should clearly describe the supported resource type, otherwise, we can't evaluate the testing is enough or not.

Reply:
fixed